keyboards/crkbd/rev1/matrix.c

```diff
@@ -179,17 +181,20 @@ int i2c_transaction(void) {
 
 #else // USE_SERIAL
 
-int serial_transaction(void) {
+int serial_transaction(int master_changed) {
     int slaveOffset = (isLeftHand) ? (ROWS_PER_HAND) : 0;
+#ifdef SERIAL_USE_MULTI_TRANSACTION
+    int ret=serial_update_buffers(master_changed);
+#else
     int ret=serial_update_buffers();
+#endif
     if (ret ) {
-        if(ret==2)RXLED1;
+        if(ret==2) RXLED1;
         return 1;
     }
-RXLED0;
-    for (int i = 0; i < ROWS_PER_HAND; ++i) {
-        matrix[slaveOffset+i] = serial_slave_buffer[i];
-    }
+    RXLED0;
+    memcpy(&matrix[slaveOffset],
+        (void *)serial_slave_buffer, sizeof(serial_slave_buffer));
```

Comment:
OIC. sizeof(serial_slave_buffer) as same as  ROWS_PER_HAND.

However, if the size to be copied to memory exceeds the size secured by the variable, it causes a bug, so I think that this part is acceptable as it is.

Therefore, I think it would be better to delete ROW_PER_HAND and use  SERIAL_SLAVE_BUFFER_LENGTH or SERIAL_MASTER_BUFFER_LENGTH.

What do you think?

Reply:
should be fine, besides `sizeof` is done during compile time not run time so it should be fairly efficient still.